content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -1231,5 +1231,5 @@ it is created.
 * `Deployment` is a top-level resource in the Kubernetes REST API.
   Read the {{< api-reference page="workload-resources/deployment-v1" >}}
   object definition to understand the API for deployments.
-* Read about [PodDisruptionBudget](/docs/concepts/workloads/pods/disruptions/) and how
-  you can use it to manage application availability during disruptions.
+* [PodDisruptionBudget](/docs/concepts/workloads/pods/disruptions/) and how you can use it to manage application availability during disruptions.
+* Using kubectl to [Create a Deployment](/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/).
```

Comment:
```suggestion
* Use kubectl to [Create a Deployment](/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/).
```

Reply:
Hi @divya-mohan0209 ,
Incorporated the suggested changes, PTAL!